src/webgpu/gpu_test.ts

```diff
@@ -1099,7 +1108,143 @@ export interface TextureTestMixinType {
     exp: PerPixelComparison<E>[],
     comparisonOptions?: TexelCompareOptions
   ): void;
+
+  /**
+   * Renders the 2 given textures to an rgba8unorm texture at the size of the
+   * specified mipLevel. Expects contents of both textures to match.
```

Comment:
nit: comment that this decompresses compressed textures (or otherwise converts source data to rgba8unorm) on the GPU, to explain why expectedTexture is a GPUTexture and not CPU-side

Reply:
done